jspredict.js

```diff
@@ -119,13 +123,17 @@
       return observes
     },
 
-    transits: function(tle, qth, start, end, minElevation) {
+    transits: function(tle, qth, start, end, minElevation, maxTransits) {
```

Comment:
Thanks for the PR! Do you mind updating the README.md file to detail this new parameter?  https://github.com/nsat/jspredict#methods